index.js

```diff
@@ -9,4 +9,11 @@
 
 server.listen(port, () => {
   console.log(`Server running on http://localhost:${port}/`);
-});
+});
+
+// Error handling
+server.on('error', (err) => {
+  console.error('Server error:', err);
+  process.exit(1);
+});
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

> Note: This review was outside of the patch, but no patch was found that overlapped with it. Original lines [20-20]

LGTM!


<!-- This is an auto-generated comment by OSS CodeRabbit -->